cms/templates/js/xblock-string-field-editor.underscore

```diff
@@ -1,16 +1,16 @@
 <div class="incontext-editor-action-wrapper">
-    <a href="" class="action-edit action-inline xblock-field-value-edit incontext-editor-open-action" title="<%- gettext('Edit the name') %>">
-        <span class="icon fa fa-pencil" aria-hidden="true"></span><span class="sr"> <%- gettext("Edit") %></span>
-    </a>
+    <button type="button" class="action-edit action-inline xblock-field-value-edit incontext-editor-open-action" title="<%- gettext('Edit the name') %>">
+        <span class="icon fa fa-pencil" aria-hidden="true"></span>
+    </button>
 </div>
 
 <div class="xblock-string-field-editor incontext-editor-form">
   <form>
     <% var formLabel = gettext("Edit %(display_name)s (required)"); %>
-    <label><span class="sr"><%= interpolate(formLabel, {display_name: fieldDisplayName}, true) %></span>
-      <input type="text" value="<%= value %>" class="xblock-field-input incontext-editor-input" data-metadata-name="<%= fieldName %>" title="<%= gettext('Edit the name') %>">
+    <label><span class="sr"><%- HtmlUtils.HTML(interpolate(formLabel, {display_name: fieldDisplayName}, true)) %></span>
```

Comment:
The goal is that those characters can be entered into the input field and display properly to the user (and any nefarious HTML being set as a display name is escaped). You should know when you look at the course outline whether or not they appear properly.

The test may need to be changed to expect .val to return something that is escaped. That's fine, as long as the text looks correct on the course outline when <, >, etc. are used.